fvm/environment/accounts.go

```diff
@@ -27,6 +27,7 @@ type Accounts interface {
 	AppendPublicKey(address flow.Address, key flow.AccountPublicKey) error
 	GetPublicKey(address flow.Address, keyIndex uint64) (flow.AccountPublicKey, error)
 	SetPublicKey(address flow.Address, keyIndex uint64, publicKey flow.AccountPublicKey) ([]byte, error)
+	GetPublicKeys(address flow.Address) ([]flow.AccountPublicKey, error)
 	GetContractNames(address flow.Address) ([]string, error)
```

Comment:
can you change the function `func (a *StatefulAccounts) GetPublicKeys(` in this file?


it wraps a returned error:
```
	count, err := a.GetPublicKeyCount(address)
	if err != nil {
		return nil, fmt.Errorf(
			"failed to get public key count of account: %w",
			err)
	}
```
 but that error is already wrapped with that exact text